docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -626,30 +617,106 @@ These steps can also be performed through BitBucket's UI wizard. This UI wizard
 
 ## Buildkite
 
-```
+To add Semgrep into your Buildkite pipeline:
+
+1. Create or edit a `pipelines.yml` configuration file to add a Semgrep command as part of your pipeline. Refer to the [BuildKite code snippet](#buildkite-code-snippet). This configuration file can also be stored within Buildkite.
+2. Copy the relevant code snippet provided after these instructions.
```

Comment:
Which code snippet? In BuildKite or below these steps? Why not include the code snippet here under step 2? Or is it the last step we have to do after we follow all instructions?

Reply:
(Done making subheaders for all CI providers.)